Java/PoolAndResourceFile/src/main/java/PoolAndResourceFile.java

```diff
@@ -173,27 +193,53 @@ private static String uploadFileToCloud(CloudBlobContainer container, String fil
         blob.upload(new FileInputStream(source), source.length());
 
         // Create policy with 1 day read permission
-        SharedAccessBlobPolicy policy = new SharedAccessBlobPolicy();
-        EnumSet<SharedAccessBlobPermissions> perEnumSet = EnumSet.of(SharedAccessBlobPermissions.READ);
-        policy.setPermissions(perEnumSet);
-
-        Calendar c = Calendar.getInstance();
-        c.setTime(new Date());
-        c.add(Calendar.DATE, 1);
-        policy.setSharedAccessExpiryTime(c.getTime());
+        SharedAccessBlobPolicy policy = getSharedAccessBlobPolicy(SharedAccessBlobPermissions.READ, 1);
 
         // Create SAS key
         String sas = blob.generateSharedAccessSignature(policy, null);
         return blob.getUri() + "?" + sas;
     }
 
+    /**
+     *
+     * @param container
+     *          container to upload the file
+     * @param filePattern
+     *          all the files matching the given pattern will be upload to container
+     * @return  list of output files
+     * @throws StorageException
+     * @throws InvalidKeyException
+     */
+    private static List<OutputFile> getOutputFiles(CloudBlobContainer container, String filePattern) throws StorageException, InvalidKeyException {
```

Comment:
nit: this is really a create or mutate function not a get.

Reply:
Also describe method in comment for sample descriptive-ness

Reply:
updated the code with suggested change.